src/test/java/org/benetech/servicenet/adapter/icarol/ICarolDataAdapterCompleteTest.java

```diff
@@ -0,0 +1,280 @@
+        assertEquals("CarpetHanger Office", result.getName());
+        assertNull(result.getAlternateName());
+        assertEquals(DIFF_LOREM_IPSUM, result.getDescription());
+        assertEquals("www.example.org", result.getUrl()); //TODO: wiki
+        assertNull(result.getEmail()); //TODO: wiki
+        assertEquals("Active", result.getStatus());
+        assertNull(result.getInterpretationServices());
+        assertEquals("Call or check website for notices on applications.", result.getApplicationProcess());
+        assertNull(result.getWaitTime());
+        assertEquals("Free", result.getFees());
+        assertNull(result.getAccreditations());
+        assertNull(result.getLicenses());
+        assertEquals("Program", result.getType());
+        assertEquals("3", result.getExternalDbId());
+    }
+    @Transactional
+
+    @Test
+    public void shouldImportCompleteLocation() {
+        adapter.importData(importData);
+        ArgumentCaptor<Location> captor = ArgumentCaptor.forClass(Location.class);
+        verify(importService, times(1))
+            .createOrUpdateLocation(captor.capture(), anyString(), anyString());
+        Location result = captor.getValue();
+
+        assertEquals("12345 Cool Street - CarpetHanger (CA)", result.getName());
+        assertEquals(40.123456, result.getLatitude());
+        assertEquals(-120.123456, result.getLongitude());
+        assertEquals("11", result.getExternalDbId());
+    }
+
+    @Test
+    @Ignore("Map county to region")
+    public void shouldImportCompletePhysicalAddress() {
+        adapter.importData(importData);
+        ArgumentCaptor<PhysicalAddress> captor = ArgumentCaptor.forClass(PhysicalAddress.class);
+        verify(importService, times(1))
+            .createOrUpdatePhysicalAddress(captor.capture(), any(Location.class));
+
+        PhysicalAddress result = captor.getValue();
+
+        assertNull(result.getAttention());
+        assertEquals("12345 Cool Street", result.getAddress1());
+        assertEquals("CarpetHanger", result.getCity());
+        assertEquals("Commoncounty", result.getRegion()); //TODO: wiki
+        assertEquals("CA", result.getStateProvince());
+        assertEquals("12345", result.getPostalCode());
+        assertEquals("United States", result.getCountry());
+    }
+
+    @Test
+    @Ignore("Map county to region")
+    public void shouldImportCompletePostalAddress() {
+        adapter.importData(importData);
+        ArgumentCaptor<PostalAddress> captor = ArgumentCaptor.forClass(PostalAddress.class);
+        verify(importService, times(1))
+            .createOrUpdatePostalAddress(captor.capture(), any(Location.class));
+
+        PostalAddress result = captor.getValue();
+
+        assertNull(result.getAttention());
+        assertEquals("12345 Cool Street", result.getAddress1());
+        assertEquals("CarpetHanger", result.getCity());
+        assertEquals("Commoncounty", result.getRegion()); //TODO: wiki
```

Comment:
I think it's better to correct the wiki now, while you know exactly what's wrong here, so in the future, when the mapping is going to be fixed, both the wiki, and expected will point out what should be the result.